packages/config/src/VitessceAutoConfig.js

```diff
@@ -337,21 +369,136 @@ function calculateCoordinates(viewsNumb) {
     const col = i % cols;
     const x = col * width;
     const y = row * height;
-    coords.push([x, y, width, height]);
+
+    // The coordinates have to be integer values:
+    coords.push([
+      Math.floor(x),
+      Math.floor(y),
+      Math.floor(width),
+      Math.floor(height),
+    ]);
   }
 
   return coords;
 }
 
-async function generateConfig(url, vc) {
+const spatialSegmentationLayerValue = {
+  radius: 65,
+  stroked: true,
+  visible: true,
+  opacity: 1,
+};
+
+function insertCoordinationSpace(views, vc) {
+  const [
+    spatialSegmentationLayer,
+    spatialImageLayer,
+    spatialZoom,
+    spatialTargetX,
+    spatialTargetY,
+  ] = vc.addCoordination(
+    ct.SPATIAL_SEGMENTATION_LAYER,
+    ct.SPATIAL_IMAGE_LAYER,
+    ct.SPATIAL_ZOOM,
+    ct.SPATIAL_TARGET_X,
+    ct.SPATIAL_TARGET_Y,
+  );
+
+  spatialSegmentationLayer.setValue(spatialSegmentationLayerValue);
+  spatialImageLayer.setValue([
+    {
+      type: 'raster',
+      index: 0,
+      colormap: null,
+      transparentColor: null,
+      opacity: 1,
+      domainType: 'Min/Max',
+      channels: [
+        {
+          selection: {
+            c: 0,
+          },
+          color: [
+            255,
+            0,
+            0,
+          ],
+          visible: true,
+          slider: [
+            0,
+            255,
+          ],
+        },
+        {
+          selection: {
+            c: 1,
+          },
+          color: [
+            0,
+            255,
+            0,
+          ],
+          visible: true,
+          slider: [
+            0,
+            255,
+          ],
+        },
+        {
+          selection: {
+            c: 2,
+          },
+          color: [
+            0,
+            0,
+            255,
+          ],
+          visible: true,
+          slider: [
+            0,
+            255,
+          ],
+        },
+      ],
+    },
+  ]);
+  spatialZoom.setValue(-2.598);
+  spatialTargetX.setValue(1008.88);
+  spatialTargetY.setValue(1004.69);
```

Comment:
cool, I am deleting them. It would be good to add a note/comment somewhere that the user should fill up their values, but I am not sure what the best place for this will be. What do you think? Or should I skip adding a comment?